drivers/bluetooth/hci/cyw43xxx.c

```diff
@@ -0,0 +1,300 @@
+/*
+ * Copyright (c) 2022 Cypress Semiconductor Corporation (an Infineon company) or
+ * an affiliate of Cypress Semiconductor Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @brief CYW43xxx HCI extension driver.
+ */
+
+#include <errno.h>
+#include <stddef.h>
+
+#include <zephyr/kernel.h>
+#include <zephyr/device.h>
+#include <zephyr/bluetooth/bluetooth.h>
+#include <zephyr/drivers/bluetooth/hci_driver.h>
+#include <zephyr/drivers/gpio.h>
+#include <zephyr/drivers/uart.h>
+
+#define BT_DBG_ENABLED IS_ENABLED(CONFIG_BT_DEBUG_HCI_DRIVER)
+#define LOG_LEVEL CONFIG_BT_HCI_DRIVER_LOG_LEVEL
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(cyw43xxx_driver);
+
+#include <stdint.h>
+
+#define HCI_UART_NODE                     DT_COMPAT_GET_ANY_STATUS_OKAY(infineon_cyw43xxx_bt_hci)
+
+/* BT settling time after power on */
+#define BT_POWER_ON_SETTLING_TIME         (500u)
```

Comment:
Use a unit suffix for these, for example `BT_POWER_ON_SETTLING_TIME_MS`. Same for the one below.